web-console/3-project-details/provisioned-services-overview.md

```diff
@@ -14,7 +14,7 @@
 - In the collapsed state, show the display name for the service class with the service instance name underneath.
 - The display name should link to a details page for that service instance.
 - If the provision service has a binding, it is listed under the label "Bindings."
-- If the service has nothing bound to it, the bindings section should be replaced by a create binding action.
+- If the service has nothing bound to it, the bindings section should be replaced by a create binding action. The pficon-add-circle-o icon should be shown in front of the "Create Binding" link.
```

Comment:
The above image does not include the circle icon mentioned here. Is that the intended appearance?